### PR TITLE
CLDR-14386 Modernize List Users in gear menu

### PR DESCRIPTION
**Pull request by btangmu**
-Add new UserList.java for back end

-Add new cldrListUsers.js and cldrAccount.js (cf. CLDR-14383) for front end

-Remove dead code

-Comments

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14386
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by btangmu** [comment]
This is far from finished. I suggest merging it with master anyway since it should cause no problems while USE_DOJO is still true, and the changes are extensive enough that we should merge to avoid merge conflicts later.

**Comment by srl295** [comment]
PR number 1,000 ! 🎉

File: tools/cldr-apps/src/main/webapp/js/new/cldrAjax.js [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * and to hide everything else, minimizing global scope pollution.
  */
 const cldrAjax = (function () {
-  const ST_AJAX_DEBUG = false;
+  const ST_AJAX_DEBUG = true;
```
Comment (srl295) [inline tools/cldr-apps/src/main/webapp/js/new/cldrAjax.js:11]: should this stay on by default?

File: tools/cldr-apps/src/main/webapp/js/new/cldrGear.js [hunk 2]
```diff
@@ -6,6 +6,13 @@
  *
  * Use an IIFE pattern to create a namespace for the public functions,
  * and to hide everything else, minimizing global scope pollution.
+ *
+ * TODO: "Avoid dropdown menus": "Usability studies show that dropdown menus are annoying";
+ * "difficult for search engines to crawl" https://www.orbitmedia.com/blog/website-navigation/
+ * Survey Tool's current dropdown gear menu is also problematic in terms of library dependencies,
+ * invalid html, inability to scroll (e.g., when console log is open and screen is small),
+ * and failure to close immediately when some items are clicked on. It would be cleaner to
+ * open a new special page ("#gear"), with a list of items, when the gear menu is clicked on.
```
Comment (srl295) [inline tools/cldr-apps/src/main/webapp/js/new/cldrGear.js:15]: @btangmu could be good, and now that the gear is a module it should be simple..

File: tools/cldr-apps/src/main/webapp/js/special/users.js [hunk 3]
```diff
@@ -69,7 +69,7 @@ define("js/special/users.js", ["js/special/SpecialPage.js"], function (
         var locked = [];
 
         var byEmail = {}; // email -> u:{}
-        var byId = {}; // id -> u:{}
+        var byId = {}; // id -> u:{} -- UNUSED!
```
Comment (srl295) [inline tools/cldr-apps/src/main/webapp/js/special/users.js:72]: drop if unused

**Review by srl295 — COMMENTED** [review]
still reviewing…

File: tools/cldr-apps/src/main/webapp/js/special/users.js (hunk 3 above)
Comment (btangmu) [reply tools/cldr-apps/src/main/webapp/js/special/users.js:72]: that whole file will be unused when we drop dojo -- I just added the comment as a reminder during the transition

File: tools/cldr-apps/src/main/webapp/js/new/cldrAjax.js (hunk 1 above)
Comment (btangmu) [reply tools/cldr-apps/src/main/webapp/js/new/cldrAjax.js:11]: it makes the console a bit noisy, so maybe we should just turn it on and off as seems most useful; I didn't notice I had switched it in this PR

File: tools/cldr-apps/src/main/webapp/js/new/cldrGear.js (hunk 2 above)
Comment (btangmu) [reply tools/cldr-apps/src/main/webapp/js/new/cldrGear.js:15]: yes

**Review by srl295 — APPROVED** [review]
RSLGTM. would be good to start using the new API mechanisms.

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java [hunk 4]
```diff
@@ -2861,6 +2880,9 @@ public static void getRow(HttpServletRequest request, HttpServletResponse respon
                 }
 
                 try {
+                    /*
+                     * TODO: why use org.json.JSONWriter here but SurveyAjax.JSONWriter elsewhere?
+                     */
```
Comment (btangmu) [inline tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java:2885]: Comment outdated by https://github.com/unicode-org/cldr/pull/1002 -- I'll fix it later

**Pull request merged**